src/test/java/DeadlineTest.java

```diff
@@ -0,0 +1,23 @@
+package test.java;
+
+import main.java.duke.Deadline;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DeadlineTest {
```

Comment:
Perhaps you could format your test methods in the following way so that it is easier to understand?
`featureUnderTest_testScenario_expectedBehavior()`